src/style.css

```diff
@@ -1 +1,64 @@
 /* Styles go here */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
```

Comment:
Don't use `*` selector for reseting default styles, reset it for particular elements